app/src/main/java/com/nevilleantony/prototype/activities/MainActivity.java

```diff
@@ -43,6 +53,21 @@ protected void onCreate(Bundle savedInstanceState) {
 		viewPagerAdapter.addFragment(SampleFragment.newInstance("Downloads Page"));
 		viewPagerAdapter.addFragment(SampleFragment.newInstance("Completed Page"));
 
+		//Downloads UI - begin
+
+		recyclerView = findViewById(R.id.recyclerView);
+		s1 = getResources().getStringArray(R.array.downloads_name);
+		s2 = getResources().getStringArray(R.array.downloads_status);
+		s3 = getResources().getStringArray(R.array.downloads_speed);
+		s4 = getResources().getStringArray(R.array.downloads_percentage);
+
+
+		RecyclerViewAdapter recyclerViewAdapter = new RecyclerViewAdapter(this,s1,s2,s3,s4);
```

Comment:
Instead of passing each properties in separate arrays, lets have a class, something like `Downloadable`. And then create an adapter that takes a `List<Downloadable>`